index.html

```diff
@@ -14,18 +14,24 @@
     <link rel="stylesheet" href="css/reset.css" />
     <link rel="stylesheet" href="css/layout.css" />
     <link rel="stylesheet" href="css/styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Proza+Libre&display=swap"
+      rel="stylesheet" />
   </head>
 
   <body>
     <header>
       <div class="container">
         <div class="row">
-          <h1>Code Journal</h1>
+          <h1 class="code-journal-header">Code Journal</h1>
+          <a>Entries</a>
```

Comment:
Needs an `href` Can just do something like `href="#"`. This will force the page to scroll to the top